Frontend/CO.CDP.OrganisationApp/Pages/ContactUs.cshtml

```diff
@@ -0,0 +1,117 @@
+                                    <span class="govuk-visually-hidden">Error:</span>
+                                    @Html.ValidationMessageFor(m => m.EmailAddress)
+                                </p>
+                            }
+                            <input class="govuk-input @(emailAddressHasError ? "govuk-input--error" : "")" id="@nameof(Model.EmailAddress)"
+                                   value="@Model.EmailAddress" name="@nameof(Model.EmailAddress)" type="text" autocomplete="email"
+                                   spellcheck="false" @(emailAddressHasError ? "aria-describedby=last-name-error" : "")>
+                        </div>
+
+                        <div class="govuk-form-group @(organisationNameHasError ? "govuk-form-group--error" : "")">
+                            <label class="govuk-label" for="@nameof(Model.OrganisationName)">
+                                @Html.DisplayNameFor(m => m.OrganisationName)
+                            </label>
+                            @if (organisationNameHasError)
+                            {
+                                <p id="first-name-error" class="govuk-error-message">
+                                    <span class="govuk-visually-hidden">Error:</span>
+                                    @Html.ValidationMessageFor(m => m.OrganisationName)
+                                </p>
+                            }
+                            <input class="govuk-input @(nameHasError ? "govuk-input--error" : "")" id="@nameof(Model.OrganisationName)"
+                                   value="@Model.OrganisationName" name="@nameof(Model.OrganisationName)" type="text"
+                                   spellcheck="false" @(nameHasError ? "aria-describedby=organisation-name-error" : "")>
+                        </div>
+
+                        <div class="govuk-form-group govuk-character-count @(messageHasError ? "govuk-form-group--error" : "")" data-module="govuk-character-count" data-maxlength="10000">
+                            <label class="govuk-label" for="@nameof(Model.Message)">
+                                Message
+                            </label>
+                            @if (messageHasError)
+                            {
+                                <p id="message-error" class="govuk-error-message">
+                                    <span class="govuk-visually-hidden">Error:</span>
+                                    @Html.ValidationMessageFor(m => m.Message)
+                                </p>
+                            }
+                            <textarea class="govuk-textarea govuk-js-character-count @(messageHasError ? "govuk-input--error" : "")"
+                                      id="@nameof(Model.Message)" name="@nameof(Model.Message)"
+                                      cols="50" rows="10" aria-describedby="@nameof(Model.Message)-info">@Model.Message</textarea>
```

Comment:
aria-describedby should contain `message-error` if there is an error _as well_ as the info hint. Space separated.

Reply:
done